websites/H/HITMAPS/presence.ts

```diff
@@ -0,0 +1,76 @@
+const presence = new Presence({
+  clientId: "842704573877714974"
+});
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "app"
+  };
+
+  if (document.location.hostname === "roulette.hitmaps.com") {
+    presenceData.details = "Playing HITMAPS™ Roulette"
```

Comment:
about this i think you can do more stuff on this subdomain

Reply:
I guess I could add one little thing... Adding that now.